effects/letter-clock/data/README

```diff
@@ -0,0 +1,11 @@
+Converting the SVG to proper PNG is as follows:
+
+Use InkScape 1.0 or newer and export PNG with antialiasing DISABLED
```

Comment:
I have major problems reproducing `clock-bg.png` from `background.svg` file. I failed to do so using method you described in this file. Most likely it's due to use of fonts that are present only in your installation of system. Please list fonts with URLs (so I can fetch them) that are required by this file.